packages/server/modules/gatekeeperCore/migrations/20250424064652_migrate_old_plans.ts

```diff
@@ -0,0 +1,37 @@
+import { Knex } from 'knex'
+
+/**
+ * The full stripe+db migration should've already executed, this is a fallback for dev/test envs to migrate broken plans

```

Comment:
we're already "sure" that nothing is on old plans, so this should be safe to have

Reply:
Why do we need this migration?
Prod should be up to date, is it for local envs?

Reply:
yes, see comment